games-engines/openmw/files/openmw-0.48.0-lua.patch

```diff
@@ -0,0 +1,20 @@
+Use the lua version selected by the lua-single eclass
+
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -400,14 +400,11 @@ if(USE_LUAJIT)
+    set(LUA_INCLUDE_DIR ${LuaJit_INCLUDE_DIR})
+    set(LUA_LIBRARIES ${LuaJit_LIBRARIES})
+ else(USE_LUAJIT)
+-   find_package(Lua REQUIRED)
++   find_package(Lua ${GENTOO_LUA_VERSION} EXACT REQUIRED)
```

Comment:
Can this bit be avoided? There's the chewi method first introduced in [slade](https://github.com/gentoo/gentoo/blob/eb34fab783d361ce64883e7bafc7a261dfadbb63/games-util/slade/slade-3.2.0_beta2.ebuild#L68) and I copied it minus the temporary variable in [meandmyshadow](https://github.com/gentoo/gentoo/blob/master/games-puzzle/meandmyshadow/meandmyshadow-0.5a.ebuild#L40)

Haven't looked closely, but it's okay if it makes things simpler considering seem still need a patch for the bit below either way. It may be easier to maintain if upstream shifts things around though.

Reply:
Used that method.